doc/developer/design/20220516_transactional_consistency.md

```diff
@@ -0,0 +1,456 @@
+# Materialize Transactional Consistency
+
+## Summary
+
+A description of Materialize’s consistency guarantees with respect to the operations that occur inside of Materialize.
+Operations include:
+
+- `SELECT`, `INSERT`, `UPDATE`, and `DELETE` statements (but not `TAIL`)
```

Comment:
tw nit: should never have one item. however, here your one item contains 4 items, so consider rewriting to not contain a list, or fully enumerate your items

Reply:
Thanks fixed